2016-03-01  Aurelien Jarno  <aurelien@aurel32.net>

	* sysdeps/unix/sysv/linux/sparc/sparc64/syscalls.list (bind, listen,
	setsockopt): Remove.

--- a/sysdeps/unix/sysv/linux/sparc/sparc64/syscalls.list
+++ b/sysdeps/unix/sysv/linux/sparc/sparc64/syscalls.list
@@ -1,10 +1,7 @@
 # File name	Caller	Syscall name	# args	Strong name	Weak names
 
-bind		-	bind		3	__bind		bind
 getpeername	-	getpeername	3	__getpeername	getpeername
 getsockname	-	getsockname	3	__getsockname	getsockname
 getsockopt	-	getsockopt	5	__getsockopt	getsockopt
-listen		-	listen		2	__listen	listen
-setsockopt	-	setsockopt	5	__setsockopt	setsockopt
 shutdown	-	shutdown	2	__shutdown	shutdown
 socketpair	-	socketpair	4	__socketpair	socketpair
